packages/extension-chakra-storefront/src/components/footer/index.jsx

```diff
@@ -9,45 +9,34 @@ import PropTypes from 'prop-types'
 import {
     Box,
     Text,
-    Divider,
+    Separator,
     SimpleGrid,
-    useMultiStyleConfig,
-    Select as ChakraSelect,
+    Group,
+    NativeSelect,
     Heading,
     Input,
-    InputGroup,
-    InputRightElement,
-    createStylesContext,
     Button,
-    FormControl
+    useSlotRecipe,
```

Comment:
Typically when imported named imports like this you order them in A-Za-z.. meaning this lowercase hook would occur at the end. Can you update these imports to be sorted properly please. 

You might want to take a look at some of the other files you have updated as well. 

(Side Note: We should probably have a lint rule for things like this.)